src/components/NewVariantTable.tsx

```diff
@@ -47,8 +47,8 @@ export const NewVariantTable = ({ country, onVariantSelect }: Props) => {
   const variants = useMemo(() => {
     const uniqueVisibleMutations = new Set();
     return data?.variants
-      .map(v => {
-        const visibleMutations = v.mutations.filter(m => {
+      .map((v: any) => {
```

Comment:
Why did you add `any` here? I've [reverted it](https://github.com/cevo-public/cov-spectrum-website/commit/171aff70d2e31892ce765d0e0fa0bf113df096b4) and it looks like the correct type gets inferred.